src/types/messages/components/button_styles.ts

```diff
@@ -0,0 +1,12 @@
+export enum ButtonStyles {
```

Comment:
```suggestion
// TODO: add docs link
export enum DiscordButtonStyles {
```
Note: Add an alias named ButtonStyles at the bottom